as2_utilities/as2_external_object_to_tf/CMakeLists.txt

```diff
@@ -0,0 +1,115 @@
+cmake_minimum_required(VERSION 3.5)
```

Comment:
Try to standardize this CMakeLists.txt with the rest of aerostack2 packages. Like behaviors motion, gazebo platform, multirotor simulator, etc

Reply:
Don't understand much this, its literally copied from our node template but with the modifications needed to define interfaces in the same node